explorer/app/components/Project/project.tsx

```diff
@@ -19,27 +21,37 @@ import {
 } from "./project.styles";
 
 interface Props {
+  tabs: { label: string }[];
   mainColumn: ReactNode;
   sideColumn: ReactNode;
   top: ReactNode;
+  tabIndex: number;
+  onTabChange: (newTab: number) => void;
 }
 
 export const Project = ({
   mainColumn,
   sideColumn,
   top,
+  tabs,
+  tabIndex,
+  onTabChange,
 }: Props): JSX.Element => {
   const tablet = useBreakpointHelper(BREAKPOINT_FN_NAME.UP, BREAKPOINT.TABLET);
   const ProjectOverview = tablet ? Overview : Overview.withComponent(FlatPaper);
   const ProjectOverviewMain = tablet ? Main : Fragment;
   const ProjectOverviewSide = tablet ? Side : Fragment;
   return (
-    <ProjectLayout>
+    <Container>
       <ProjectHero>{top}</ProjectHero>
-      <ProjectOverview>
-        <ProjectOverviewMain>{mainColumn}</ProjectOverviewMain>
-        <ProjectOverviewSide>{sideColumn}</ProjectOverviewSide>
-      </ProjectOverview>
-    </ProjectLayout>
+      <Tabs onTabChange={onTabChange} selectedTab={tabIndex} tabs={tabs}>
```

Comment:
Apologies for the lack of communication in this area Bruce; it looks like Fran has updated the structure of "hero" section of the project detail page and has a slot for the tab functionality [here](https://github.com/clevercanary/data-browser/blob/detail-tabs-%23120/explorer/app/components/Project/components/Hero/hero.tsx#L39). Would you be able to take a look at moving `<Tabs>` into this slot?

Reply:
Sure thing!